### PR TITLE
Add BE/BS fields to the WebauthnDevice proto

### PR DESCRIPTION
**Pull request by codingllama**
Add fields to track the BE (Backup Eligibility) and BS (Backup State) WebAuthn credential flags.

Follow up from #45029.

### CODE REVIEW

File: buf-gogo.gen.yaml [hunk 1]
```diff
@@ -2,4 +2,6 @@ version: v1
 plugins:
   - name: gogofast
     out: ./gogogen
-    opt: plugins=grpc
+    opt:
+      - plugins=grpc
+      - Mgoogle/protobuf/wrappers.proto=github.com/gogo/protobuf/types
```
Comment (codingllama) [inline buf-gogo.gen.yaml:7]: This makes sure we use the gogo-backed types for wrappers.proto. We "lucked out" in the current usage, as events.proto sets the options to turn the fields into pointers instead.

**Review by probakowski — APPROVED** [review]

**Review by rosstimothy — APPROVED** [review]

**Review by AntonAM — APPROVED** [review]

**Comment by codingllama** [comment]
Thanks everyone. Terraform docs updated at 0a8cba60b9800f378396540229c2a058beee56fd.

**Comment by codingllama** [comment]
Not sure what's up with the terraform docs, codegen seems to output invalid links but it's only following the template. @hugoShaka, any ideas?

**Comment by hugoShaka** [comment]
Blocked by https://github.com/gravitational/teleport/pull/45175

**Comment by codingllama** [comment]
Rebased onto master, should merge cleanly now.

**Pull request merged**